docs/src/pages/index.tsx

```diff
@@ -167,7 +167,7 @@ print(image_query)`;
                   <div className={baseStyles.integrationcompatibilityitem}>
                     <h3 className={baseStyles.title}>API</h3>
                     <p>API to let you access your models in the cloud - no need to run your own models or hardware.</p>
-                    <a href="/python-sdk/docs/getting-started " className={`${baseStyles.cmnButton} ${baseStyles.outline}`}>Learn More</a>
+                    <a href="/python-sdk/docs/api-reference-docs" className={`${baseStyles.cmnButton} ${baseStyles.outline}`}>Learn More</a>
```

Comment:
Wait, is this right? Is it not `/python-sdk/apireference-docs` (without docs)

Reply:
Whoops! Fixed